src/olympia/blocklist/tests/test_admin.py

```diff
@@ -1196,6 +1208,7 @@ def test_can_not_edit_without_permission(self):
         # Try to edit the block anyway
         response = self.client.post(
             self.change_url, {
+                'input_guids': self.block.guid,
```

Comment:
Why was this change necessary in this patch?

Reply:
It wasn't strictly necessary but I noticed it so drive-by fixed it.  The `.post` would still fail anyway but if the permission checks were broken in subsequent patch then, without `input_guids`, it would fail in a different way (it'd 200 back to the same page because the form data was broken) which could be confusing.